src/litdata/processing/data_processor.py

```diff
@@ -30,16 +30,16 @@
 from typing import Any, Dict, List, Optional, Tuple, TypeVar, Union
 from urllib import parse
 
+import boto3
+import botocore
```

Comment:
Note that boto3 (including botocore) are core dependencies (requirements.txt) so we don't need a conditional import as before.